data/battle_anim_scripts.s

```diff
@@ -33208,7 +33660,7 @@ FINISH_SOULBLAZE:
 	call ResetFromWhiteScreen
 	blendoff
 	createsprite gSlideMonToOriginalPosSpriteTemplate, ANIM_ATTACKER, 2, ANIM_TARGET, 0x0, 0x10
-	createsprite gSlideMonToOriginalPosSpriteTemplate, ANIM_ATTACKER, 2, ANIM_DEF_PARTNER, 0x0, 0x10
+	createsprite gSlideMonToOriginalPosPartnerSpriteTemplate, ANIM_ATTACKER, 2, ANIM_DEF_PARTNER, 0x0, 0x10
 	waitforvisualfinish
 	end
 ClangorousSoulblazeEnergySwirl:
```

Comment:
Something is broken with Clangorous Soulblaze in doubles
![pokeemerald-test-3](https://github.com/user-attachments/assets/311157ba-441c-42d2-bb02-c6dc6a21c879)

Not commenting on hex numbers, since stuff might change in this

Reply:
It's still broken for me

Reply:
I run it through the testing system.
I write a simple test that just uses the moves.